tests/PasswordCheckerTest.php

```diff
@@ -40,6 +40,24 @@ public function test_fails_due_to_short_password(): void
         $this->checker->validate('abc');
     }
 
+    public function test_fails_due_to_customized_length(): void
+    {
+        $this->checker->setMinLength(14);
+
+        $this->expectException(PasswordException::class);
+        $this->expectExceptionMessage('New password must be at least 14 characters long');
+
+        $this->checker->validate('canyouhearme1');
+    }
+
+    public function test_fails_due_to_character_requirements(): void
```

Comment:
Maybe we should test this with a data provider to ensure all the requirement checks work?

Reply:
good idea